plugins/synaptics-mst/fu-synaptics-mst.rs

```diff
@@ -3,12 +3,12 @@
 
 #[derive(ToString)]
 enum SynapticsMstFamily {
-    Unknown,
-    Tesla,
-    Leaf,
-    Panamera,
-    Cayenne,
-    Spyder,
+    Unknown = 0xFF,
+    Tesla = 0,
+    Leaf = 1,
+    Panamera = 2,
```

Comment:
@CloudExt I have no idea if this is true -- I wondered if generation 1 should really be `TeslaLeaf` or something.

Reply:
That's correct.

Reply:
@CloudExt as in the code I wrote is correct? or TeslaLeaf? Thanks!

Reply:
Your code is correct, we have two old product lines Tesla and Leaf.